docs/source/workflows/base/index.rst

```diff
@@ -3,6 +3,8 @@
 Base workflows
 ==============
 
+Workflows for basic material properties that define a common interface and are implemented for various quantum engines.
```

Comment:
```suggestion
Workflows for performing basic operations or computing basic material properties that define a common interface and are implemented for various quantum engines.
```

Reply:
I added this because a structure optimization is not really a material property as say a band gap is.